dune-project

```
@@ -25,7 +25,15 @@
  (name whilloc)
  (synopsis "A short synopsis")
  (description "A longer description")
- (depends ocaml dune z3 menhir bos cmdliner ppx_deriving_yojson)
+ (depends 
+   ocaml 
+   dune 
+   z3 
+   menhir 
+   bos 
+   cmdliner 
+   ppx_deriving_yojson
+   (encoding (>= "0.0.4")))
```

```suggestion
   (encoding (= "dev")))
```

If this works we should add it